pmm-app/src/pmm-qan/panel/components/Filters/Filters.styles.ts

```diff
@@ -9,39 +9,42 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
 
   return {
     getFiltersWrapper: (height) => css`
-    border: 1px solid rgb(40, 40, 40);
-    height: ${height}px;
-    padding: 10px 16px !important;
-  `,
+      border: 1px solid rgb(40, 40, 40);
```

Comment:
Some values in this file can be replaced with theme variables (and wonder if we need `!important` on some of this styles).

Reply:
thanks for noticing, we have a lot of styles code in QAN and it will be refactored very soon